packages/timeline-state-resolver/src/integrations/vmix/index.ts

```diff
@@ -1454,3 +1432,127 @@ export interface VMixAudioChannel {
 	meterF2: number
 	headphonesVolume: number
 }
+
+/**
+ * Handles pre-loading of inputs.
+ */
+class VMixInputHandler extends EventEmitter {
+	// how long to let an input linger before removing it
+	private TTL = 1000
```

Comment:
This is already an improvement, but since we have some logic in `_resolveInputsState` that is able to reorder commands based on whether the input is (was?) on air or not, perhaps we could also use that to delay the removal by at least the duration of the transition. That's for another PR of course